packages/labs/react/src/create-component.ts

```diff
@@ -381,14 +382,13 @@ the element.`);
     }
   }
 
-  const ForwardedComponent: ReactWebComponent<I, E> = React.forwardRef<
+  const ForwardedComponent: ReactWebComponent<I, E> = ReactModule.forwardRef<
     I,
     WebComponentProps<I, E>
   >((props, __forwardedRef) =>
     createElement<Props, ReactComponent, typeof ReactComponent>(
       ReactComponent,
-      {...props, __forwardedRef},
-      props?.children
```

Comment:
This was removed as it's unnecessary and also causing a type error.

Reply:
For my own understanding, why was it provided originally?

Reply:
The actual `createElement()` signature is that children should be passed in as separate arguments after props. This is how JSX transforms would do it. But in component code, `children` is provided as part of the props object. So understandably users often do the same when calling `createElement()`. The actual implementation will use the `children` provided in props, unless a 3rd argument is provided in which case it overwrites what's provided in props. https://github.com/facebook/react/blob/v18.2.0/packages/react/src/ReactElement.js#L401-L417

Interestingly this was discussed before here https://github.com/lit/lit/pull/1420#discussion_r541337739 and was agreed to remove there but this was a missed spot.

Reply:
Awesome – makes sense! Thank you!